init-script/semi_sync_run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+rm -rf /var/lib/mysql/lost+found
```

Comment:
As data directory has changed, `lost+found` should be deleted from that directory too.
Add `rm -rf /var/lib/mysql/data/lost+found`